hztb-rest-api/src/main/java/com/svs/hztb/api/sm/model/group/GroupOutput.java

```diff
@@ -13,6 +13,8 @@
 		private List<GroupDetail> groupDetailList = new ArrayList<GroupDetail>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS2293)